public/data/ckvignesh.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "C K Vignesh",
+  "bio": "Programmer",
+  "avatar": "https://github.com/ckvignesh.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ckvignesh",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vigneshCodes",
+      "icon": "twitter"
+    },
+    {
+        "name": "My Open Source Contributions",
+        "url": "https://ckvignesh.notion.site/List-of-Open-Source-Contributions-09b2368ec25742a0b9f0421a0d1c9f0c",
+        "icon": "github"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Tech Blogging",
+      "date": "October 2021",
+      "icon": "dollar",
+      "color": "black",
+      "description": "Sharing my journey through #100DaysOfCode",
+      "url": "https://hashnode.ckvignesh.dev"
+    },
+    {
+      "title": "Started #100DaysOfCode",
+      "image": "https://github.com/ckvignesh.png",
+      "date": "25 October 2021",
+      "icon": "github",
+      "color": "red",
+      "description": "Started this journey to upskill myself and document my learning",
+      "url": "https://twitter.com/vigneshCodes"
+    }
+  ]
+}
```

Comment:
```suggestion
}

```

Reply:
Will do.

Reply:
Isn't this the closing [] of milestones?
![GT](https://user-images.githubusercontent.com/61013244/140947725-831872d7-0699-4ad6-b208-b46992e9bc30.png)

Reply:
Thanks, @krishguptadev